src/praisonai-agents/praisonaiagents/agent/agent.py

```
@@ -4859,28 +4881,138 @@ async def handoff_to_async(
+        attached; otherwise this is a no-op so the default (journal-off) path
+        stays zero-overhead.
+        """
+        journal = getattr(self, "_run_journal", None)
+        record = getattr(journal, "record", None)
+        if journal is None or not callable(record):
+            return
+        try:
+            for payload in self._verification_journal_payloads(results):
+                record(kind="verification", data=payload)
+        except Exception:  # pragma: no cover - journalling is best-effort
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)